### PR TITLE
Style and typo improvements

### PR DESCRIPTION
**Pull request by mlabod**
- Better positioning for the sidebar
- Only show sidebar scrollbar when needed
- Fixed typos
- Correct apostrophes

### CODE REVIEW

File: content/01-docs/03-pages-and-urls/page.txt [hunk 1]
```diff
@@ -32,7 +32,7 @@ Pretty obvious, this one. [The one and only Markdown](https://daringfireball.net
 
 ### Handlebars
 
-Using Handlebars syntax, you can include another value from the page textfile.
+Using Handlebar′s syntax, you can include another value from the page textfile.
```
Comment (adamrenklint) [inline content/01-docs/03-pages-and-urls/page.txt]: Handlebars is the name. Maybe it should be "Handlebars' syntax"?

Comment (mlabod) [inline content/01-docs/03-pages-and-urls/page.txt]: But the apostrophe at the end is only there if it's plural, right? 
http://www.grammar.cl/rules/genitive-case.htm

Comment (adamrenklint) [inline content/01-docs/03-pages-and-urls/page.txt]: Using the Handlebars syntax

**Comment by mlabod** [comment]
ready to merge? @adamrenklint

**Comment by adamrenklint** [comment]
danke!

**Pull request merged**